packages/firestore/karma.conf.js

```diff
@@ -0,0 +1,50 @@
+/**
+* Copyright 2017 Google Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+const karma = require('karma');
+const path = require('path');
+const karmaBase = require('../../config/karma.base');
+const { argv } = require('yargs');
+
+const testFiles = (files => {
+  if (!files) return;
+  if (Array.isArray(files)) return files.map(file => ({ pattern: file }));
+  return [{ pattern: files }];
+})(argv.testFiles);
+
+module.exports = function(config) {
+  const karmaConfig = Object.assign({}, karmaBase, {
+    // files to load into karma
+    files: testFiles
+      ? testFiles
+      : [
+          { pattern: `test/unit/bootstrap.ts` },
+          { pattern: `test/integration/bootstrap.ts` }
+        ],
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    client: {
+      mocha: {
+        timeout: 5000,
```

Comment:
Lets go ahead and increase this to 20000. I am seeing intermittent timeouts too, and I think 5 seconds may realistically be too slow when running against prod.

Reply:
Maybe get rid of "retries: 3" though...  I don't want to hide flaky tests, and I think this makes it weird / confusing to debug tests.

Reply:
Sounds good!